encode.go

```diff
@@ -913,7 +914,13 @@ func (e *encodeState) string(s string, escapeHTML bool) int {
 				e.WriteString(s[start:i])
 			}
 			switch b {
-			case '\\', '"':
+			case '\\':
+				if i+1 < l && s[i+1] == 'u' {
```

Comment:
This is wrong, we need to escape things C# normally escapes, but this just breaks serialization (I can legitimately have "\uxxxx" string that I want to be correctly JSONized and passed to other party).